astropy/cosmology/_io/tests/test_latex.py

```diff
@@ -59,9 +59,13 @@ def test_write_latex_unsupported_format(self, write, tmp_path):
         fp = tmp_path / "test_write_latex_unsupported_format.tex"
         invalid_format = "unsupported"
         with pytest.raises((ValueError, IORegistryError)) as exc_info:
-            pytest.raises(ValueError, match="format must be 'ascii.latex'")
-            pytest.raises(IORegistryError, match="No writer defined for format")
             write(fp, format=invalid_format)
+        assert (
+            exc_info.type is ValueError
+            and "format must be 'ascii.latex'" in exc_info.value.args[0]
+            or exc_info.type is IORegistryError
+            and "No writer defined for format" in exc_info.value.args[0]
+        )
```

Comment:
It can raise either error. Removing one or the other from the with statment produces failing tests. Maybe there is a better way to combine them in the with raises statement but I haven't been able to come up with one.

Reply:
It might be the case that the test needs to be redesigned, but that task is beyond the scope of this pull request.